MotionPlanningEnv/sphereObstacle.py

```diff
@@ -138,18 +141,17 @@ def add2Bullet(self, pybullet):
         else:
             raise DimensionNotSuitableForEnv("Pybullet only supports three dimensional obstacles")
         collisionShape = pybullet.createCollisionShape(pybullet.GEOM_SPHERE, radius=self.radius())
-        visualShapeId = -1
+        visualShapeId = self.id()
         baseOrientation = [0, 0, 0, 1]
-        mass = int(self.movable())
         pybullet.setAdditionalSearchPath(os.path.dirname(os.path.realpath(__file__)))
+
         visualShapeId = pybullet.createVisualShape(
             pybullet.GEOM_MESH,
-            fileName='sphere_smooth.obj',
-            rgbaColor=[1.0, 0.0, 0.0, 1.0],
-            specularColor=[1.0, 0.5, 0.5],
+            fileName='sphere.obj',
+            rgbaColor=self.color(),
             meshScale=[self.radius(), self.radius(), self.radius()]
         )
-        pybullet.createMultiBody(mass,
+        pybullet.createMultiBody(self.mass(),
```

Comment:
I will have to check if it breaks the movable feature. Because the default value is changed to `1` and `-1` would be result in a fixed obstacle.

Reply:
A negative mass is not allowed any more. 
side note: any mass which is 0 or smaller than 0 makes the obstacle fixed